src/utils/hashes.c

```diff
@@ -18,7 +18,7 @@ struct nrm_hash_s {
 	UT_hash_handle hh;
 };
 
-struct nrm_hash_iterator_s {
+struct nrm_hash_iterator_t {
```

Comment:
I believe this struct definition is not needed anymore

Reply:
Wrong file?

Reply:
No, I mean this specific struct definition is not needed at all, it can be removed